pkg/cmd/admin/policy/reconcile_sccs.go

```diff
@@ -0,0 +1,241 @@
+package policy
+
+import (
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kapierrors "k8s.io/kubernetes/pkg/api/errors"
+	kclient "k8s.io/kubernetes/pkg/client/unversioned"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+
+	"github.com/openshift/origin/pkg/cmd/server/bootstrappolicy"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	"k8s.io/kubernetes/pkg/util/sets"
+)
+
+// ReconcileSCCRecommendedName is the recommended command name
+const ReconcileSCCRecommendedName = "reconcile-sccs"
+
+type ReconcileSCCOptions struct {
+	Confirmed bool
```

Comment:
> SCC subjects, replace

this one is a little tricky.  We use the ensure method to craft additional SA user names to add to the SCC and replacing the groups/users would squash that.  I'll see if there is a good way to do it.  The other pieces seem easy to add flags for.